### DISCUSSION TITLE
相关搜索URL能和地址栏的保持一致吗？

### DISCUSSION THREAD

**Discussion by ahaitodo** [gaboolic/cloudflare-reverse-fast#18] [General]
作者您好：
假如：
https://www.url.cn/proxy/www.google.com/search?q=notepad%20%20%20%20
相关搜索search?的URL，能不能和地址栏的保持一致
https://www.url.cn/search?sca_esv=593928490&q=notepad%2B%2B+mac&sa=X&ved=2ahUKEwioi7uijK-DAxWi2gIHHXOwB30Q1QJ6BAgqEAE
 要是这个问题能解决就完美了。

**Comment by gaboolic** [comment]
你把域名部分写死，就可以了
